source/Line.cpp

```diff
@@ -1326,13 +1342,25 @@ void Line::Output(double time)
 				}
 			}
 			
-			*outfile << "\n";
+			if (env->outputMode != 1)
+				*outfile << "\n";
 		}
 		else cout << "Unable to write to output file " << endl;
 	}
 	return;
 };
 
+ofstream* Line::getOutputFilestream()
+{
+	return outfile;
```

Comment:
We should really stop doing this... It is awful :-p

We must start using std::move